plugins/chat/spec/requests/chat/api/channel_threads_controller_spec.rb

```diff
@@ -260,7 +260,7 @@
     context "when channel does not exist" do
       it "returns 404" do
         channel_1.destroy!
-        post "/chat/api/channels/#{channel_id}", params: params
+        post "/chat/api/channels/#{channel_id}/threads", params: params
```

Comment:
We weren’t calling the proper route. Instead, we were calling a non-existent route, so this example was passing anyway.